src/portable/renesas/rusb2/dcd_rusb2.c

```diff
@@ -672,6 +739,9 @@ bool dcd_edpt_open(uint8_t rhport, tusb_desc_endpoint_t const * ep_desc)
 
   /* setup pipe */
   dcd_int_disable(rhport);
+  #if (CFG_TUSB_RHPORT1_MODE & OPT_MODE_DEVICE)
+  RUSB2->PIPEBUF = 0x7C08;
```

Comment:
You are right, that hardcoded value is very strange; I asked a colleague and it was introduced to fix mass storage emulation by copying it from fsp sources (where it was hardcoded at the time but since I can't find it anymore probably Renesas could have fixed the surrounding code :slightly_smiling_face: )
To test if it doesn't regress we could use some of the MSC examples in the arduino core

Reply:
Thanks for the explanation. Due to short of time, we will keep this magic as it is. And probably with a bit of comment for refactoring later on :)